README.rst

```diff
@@ -171,13 +215,6 @@ The following will embed a CARTO map in a Jupyter notebook, allowing for custom
 
 .. image:: https://raw.githubusercontent.com/CartoDB/cartoframes/master/docs/img/map_demo.gif
 
-.. note::
-    Legends are under active development. See
-    https://github.com/CartoDB/cartoframes/pull/184 for more information. To
-    try out that code, install `cartoframes` as:
-
-        `pip install git+https://github.com/cartodb/cartoframes.git@add-legends-v1#egg=cartoframes`
-
 Data Observatory
 ----------------
```

Comment:
We can move the `Credentials` class to `cartoframes.auth` so we can update the section `CARTO Credential Management`

Reply:
👍 I don't mind doing that in this PR but seems like it might be a good idea to do in another one.  Just created an issue: https://github.com/CartoDB/cartoframes/issues/795

Reply:
True